modules/stats-widget/src/stats-widget.js

```diff
@@ -90,6 +90,9 @@ export default class StatsWidget {
   }
 
   _createDOM(container) {
+    if (!document) {
```

Comment:
This could crash if document is not defined.

`if (typeof document === 'undefined' || !document)` may be safer